tests/unit/lms/resources/_lti_launch_test.py

```diff
@@ -366,29 +364,35 @@ def test_hypothesis_config_raises_if_theres_no_tool_consumer_instance_guid(
             HTTPBadRequest,
             match='Required parameter "tool_consumer_instance_guid" missing from LTI params',
         ):
-            LTILaunchResource(pyramid_request).hypothesis_config
+            LTILaunchResource(pyramid_request).js_config
 
-    def test_hypothesis_config_contains_one_service_config(self, lti_launch):
-        assert len(lti_launch.hypothesis_config["services"]) == 1
+    def test_js_config_contains_one_service_config_for_the_client(self, lti_launch):
+        assert len(lti_launch.js_config["hypothesisClient"]["services"]) == 1
```

Comment:
Are we getting to the point here were a separate test class to hold "hypothesisConfig" related tests might be nice? 

* It allows you to collapse some if you aren't interested in them 
* Facilitates shorter names for everything (In "TestHypothesisConfig" we can use "test_it_blah")
* Could then some helper methods, or a 'client_config' fixture etc?
* Stages us nicely for the future if these get plucked out of the class to a service

Reply:
I'm probably gonna skip this one for now because almost the very next thing to do is cut all this code and its related tests, and move it all into a service. If the tests are still too long at that point then I might split them into separate test classes